python/interpret_community/mimic/model_distill.py

```diff
@@ -76,5 +77,6 @@ def _model_distill(teacher_model_predict_fn, uninitialized_surrogate_model, data
         # Use hard labels for regression or multiclass case
         if training_labels is None:
             training_labels = teacher_y
+        import pdb; pdb.set_trace()
```

Comment:
Same as above to what @gregorybchris alluded to for set_trace()

Reply:
yes this should be removed, good catch!